README.md

```diff
@@ -1,131 +1,20 @@
-[![Build Status](https://travis-ci.com/TheAlgorithms/C.svg?branch=master)](https://travis-ci.com/TheAlgorithms/C)  
-C
-========
+# The Algorithms - C # {#mainpage}
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/kvedala/C) 
+[![Gitter chat](https://img.shields.io/badge/Chat-Gitter-ff69b4.svg?label=Chat&logo=gitter&style=flat-square)](https://gitter.im/TheAlgorithms)
+[![contributions welcome](https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=0059b3&style=flat-square)](https://github.com/kvedala/C-Plus-Plus/blob/master/CONTRIBUTING.md)&nbsp;
+![GitHub repo size](https://img.shields.io/github/repo-size/kvedala/C-Plus-Plus?color=red&style=flat-square)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed/kvedala/C?color=green&style=flat-square)
+![Doxygen CI](https://github.com/kvedala/C/workflows/Doxygen%20CI/badge.svg)
+![Awesome CI Workflow](https://github.com/kvedala/C/workflows/Awesome%20CI%20Workflow/badge.svg)
```

Comment:
All these badges are pointing to your repository.
Shouldn't it point to the `TheAlgorithms`?

Reply:
Yes, I will change them once the PR is ready to be merged. Thats what I did in the C++ repository as well